monitoring/grafana-datasource-es.yaml

```diff
@@ -4,8 +4,8 @@ datasources:
 - name: Elastic
   type: elasticsearch
   access: proxy
-  database: kubernetes_cluster-*
-  url: https://v4m-es-client-service.logging:9200
+  database: viya_logs-*
+  url: https://v4m-es-client-service.__logNS__:9200
   basicAuth: true
   basicAuthUser: admin
```

Comment:
As we talked about, using the "admin" user for now (while this is all experimental) is fine...but eventually we need to move to use a different user: either the logadm user or a custom user created for this purpose.  And, we probably want to look at pulling this out into a stand-alone script so we can do this for the tenant admins  as well.

Reply:
Good point on the tenant admins. We haven't talked about making logs available for the tenant instances of Grafana. That's probably a separate story.

If `logadm` is appropriate here, should we just switch to that?  I think what we really need is a read-only user.

Reply:
Using 'logadm' would be better than using 'admin'.  However, we do not persist the 'logadm' password anywhere so we can automatically get it.  We could create a new user for this purpose.  We could even do that programmatically (we have functions for doing it) in this code or in a stand-alone script version of this code.  A stand-alone script for this could (should) allow for namespace, tenant, user and password to be passed in but also make intelligent default choices for those.